src/content/en/tools/chrome-devtools/console/command-line-reference.md

```diff
@@ -102,6 +111,17 @@ displays the value of each element's `src` property:
 ![Example of using $$() to select all images in the document and display their
 sources.](images/all-selector.png)
 
+This function also supports a second parameter, startNode, that specifies an element or Node from which to search for elements. The default value of this parameter is `document`. 
+
+This modified version of the previous example uses `$$()` to create an array of all `<img>` elements that appear in the current document after the selected Node:
+
+    var images = $$('img', document.querySelector('.devsite-header-background'));
+       for (each in images) {
+           console.log(images[each].src);
+       }
+
+![Example of using $$() to select all images appearing after the select div element in the document and displaying their sources.](images/all-selector-div.png)
+
 Note: Press <kbd class='kbd'>Shift</kbd> + <kbd class='kbd'>Enter</kbd> in the console to start
 a new line without executing the script.
 
```

Comment:
On the section title please add the optional second parameter.

    ##  $x(path, [startNode]) {: #xpath }

Reply:
Also do this for the section titles of `$` and `$$`